src/index.html

```diff
@@ -22,6 +22,375 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        class="image"
+        src="././images/logo.png"
+        alt="Moyo logo"
+      />
+      <nav class="navigation">
+        <ul class="list">
+          <li class="list__item">
+            <a
+              href="#"
+              class="list__link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="list__item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="list__item">
```